test/bdd/pkg/context/context.go

```diff
@@ -6,17 +6,25 @@ SPDX-License-Identifier: Apache-2.0
 
 package context
 
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/packer/legacy/authcrypt"
+
+	"github.com/trustbloc/edv/pkg/restapi/edv/operation"
+)
+
 // BDDContext is a global context shared between different test suites in bddtests
 type BDDContext struct {
-	EDVHostURL string
-	Args       map[string]string
+	EDVHostURL                 string
+	Packer                     *authcrypt.Packer
```

Comment:
add a TODO to replace with jwe document instead of legacy/authcrypt

Reply:
Done. Also added a ToDo above the import statement in context.go